src/SFA.DAS.EpaoRegister/InnerApi/Requests/GetEpaoRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EpaoRegister.InnerApi.Requests
+{
+    public class GetEpaoRequest : IGetApiRequest
+    {
+        public string EpaoId { get; set; }
```

Comment:
not sure we do everywhere but it would probably be better to have this set in the constructor